### ISSUE TITLE
loss = loss.mean() Is this code redundant？

### ISSUE THREAD

**Issue by Sdy344** [xingyizhou/CenterNet#787]
i have a question
/src/lib/trains/base_trainer.py
loss = loss.mean()   Calculate the average loss of batch
but
/src/lib/models/losses.py
loss = loss / (mask.sum() + 1e-4)  Calculate the average loss of all targets in the batch
so，loss = loss.mean() Is this code redundant？
